CRM/CiviAwards/Setup/AddAwardsCgExtendsOptionValue.php

```diff
@@ -33,4 +38,28 @@ public function apply() {
     ]);
   }
 
+  /**
+   * Enables/Disables the Awards option values.
```

Comment:
Can you add comment on the description as to why we are changing the description field also?